cmd/meroxa/root/connectors/create.go

```diff
@@ -93,29 +93,25 @@ func (c *Create) CreateConnector(ctx context.Context) (*meroxa.Connector, error)
 		return nil, fmt.Errorf("could not parse metadata: %w", err)
 	}
 
-	// merge in input
-	if c.flags.Input != "" {
-		config["input"] = c.flags.Input
+	if c.flags.Pipeline == "" {
```

Comment:
Checking the Pipeline flag was about 20 lines below, and I just moved it up. I don't remember why I moved it, maybe it doesn't need to move at all. I don't know why it's checked explicitly and not handled by the required notation. Since input is no longer directly stored in the config map and only assigned to the type field on new line 133, I removed the code that checks if it's non-empty and assigns it to the map.